.github/workflows/Image.yaml

```diff
@@ -223,7 +183,7 @@ jobs:
     with:
       oci-image-name: "${{ matrix.name }}_${{ matrix.commit }}_${{ matrix.revision }}"
       oci-image-path: "oci/${{ matrix.name }}"
-      is-a-rock: ${{ matrix.dockerfile-build == '' && true || false }}
+      is-a-rock: true
```

Comment:
well, if this is always true, we might as well get rid of it and assume it is true